### PR TITLE
Nicky/wd 61 rest library

### PR DESCRIPTION
**Pull request by bonellon**
Added Non-runnable REST Library.

ToDo: Do we need to write tests? How??

### CODE REVIEW

**Review by ats0stv — CHANGES_REQUESTED** [review]
Looks all good.
Just could you please remove the temp file like .project, /bin etc

**Comment by bonellon** [comment]
PR Updated - removed temp files

**Review by ats0stv — DISMISSED** [review]

File: Code/Libraries/RESTLibrary/src/main/java/ie/tcd/wayfinders/restLibrary/Test.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package ie.tcd.wayfinders.restLibrary;
```
Comment (GeeseHoward) [inline Code/Libraries/RESTLibrary/src/main/java/ie/tcd/wayfinders/restLibrary/Test.java]: Remove temp class

File: Code/Libraries/RESTLibrary/src/test/java/ie/tcd/wayfinders/restLibrary/LibraryTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import ie.tcd.wayfinders.restLibrary.Library;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class LibraryTest {
+
+    @Test
+    public void get_200OK_noProperties() throws IOException
```
Comment (GeeseHoward) [inline Code/Libraries/RESTLibrary/src/test/java/ie/tcd/wayfinders/restLibrary/LibraryTest.java]: Conform to camel case

**Comment by bonellon** [comment]
@GeeseHoward  fixed comments, please confirm

**Review by GrennanM — DISMISSED** [review]

**Review by zihan0 — APPROVED** [review]

**Review by Jamesxu182 — APPROVED** [review]

**Pull request merged**